backend/src/database/migrations/20230216082014-update-transactional-sms.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const {MessageStatus} = require('@core/constants');
+const {TransactionalSmsMessageStatus} = require('@sms/models');
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.addColumn('sms_messages_transactional', 'status', {
+            type: Sequelize.DataTypes.ENUM(
+                Object.values(TransactionalSmsMessageStatus)
+            ),
+            allowNull: false,
+            defaultValue: TransactionalSmsMessageStatus.Delivered
```

Comment:
`defaultValue` should be `UNSENT`

Reply:
the issue with defaultValue unsent in migration script would be that on column creation, the old records would all be unsent. Then i would have to run another update script to update to delivered. currently i init the column to unsent when i create the record

Reply:
prod has 724 records, if we are okay with running raw sql in prod to manually update to delivered, i can change the migration script to default to unsent, what do you think? @stanleynguyen

Reply:
> prod has 724 records, if we are okay with running raw sql in prod to manually update to delivered, i can change the migration script to default to unsent, what do you think? @stanleynguyen

I dont think we need to touch the existing records, just need to update the current column definition manually to have `UNSENT` as default and also update the js migration file (without running it)

Reply:
hmmm status is a new column, so if I set default value as unsent, it'll automatically set all previous records status to unsent.

Reply:
wait hang on, I think we're confusing ourselves here. Can huddle?

Reply:
Eh it just clicked for me what u're talking about. U're concerned that users might see historical records in `UNSENT` state and wonder why right? I think that's ok, if they ask we will can provide the information that we only migrated on the deployment date so all historical messages prior to that are defaulted to UNSENT

Reply:
another alternative is to make the column nullable and make historical messages null.

we should probably should note this behavior somewhere obvious (in the data model?) in case someone else comes along and runs some SQL analysis giving them weird results

Reply:
aaahaha yes! ok if there's no problem then i'll set it to default unsent

Reply:
done